src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/slm-implementation-guide.mdx

```diff
@@ -7,8 +7,12 @@ tags:
   - Implementation guide
 metaDescription: Service Level Management's overall goal is to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders.
 redirects:
+  - /docs/1-establish-objectives-baselines-define-team-slos
+  - /docs/1-adopt-easily-establish-objectives-baselines-define-team-slos
+  - /docs/establish-objectives-baselines-define-team-slos
   - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/service-level-management-tutorial/
   - /docs/new-relic-solutions/observability-maturity/slm-implementation-guide/
+  - /docs/using-new-relic/welcome-new-relic/optimize-your-cloud-native-environment/establish-objectives-baselines-define-team-slos/
```

Comment:
@rhetoric101 I think if we remove this line it will get rid of some of those warnings

Reply:
@clarkmcadoo, I can remove this, but I think the user is trying to ensure that if someone goes to /docs/using-new-relic/welcome-new-relic/optimize-your-cloud-native-environment/establish-objectives-baselines-define-team-slos/ that it will be redirected to src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/slm-implementation-guide.mdx. 

Also, that error in the console is confusing since it says that "There are routes that match both page and redirect." That's not the case here is it?

Reply:
There's a page associated file path, I think if we no longer want that page on the site we'd want to remove that `.mdx` file then?

Reply:
To close the loop on this, the issue was that the old file was never deleted, which was causing the console error. The original doc was still there and the redirect for that doc was also there, and this caused the issue. I removed the old file per the SME.